dev/benchmarking/ddl/1_tables.sql

```diff
@@ -0,0 +1,291 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+*/
+
+CREATE EXTENSION IF NOT EXISTS "citext";  -- type for case-insensitive text
+
+-- CPU
+CREATE TABLE IF NOT EXISTS public.cpu
+(
+  id SERIAL PRIMARY KEY
```

Comment:
While redundant, in the past, I have found it useful to prefix ID with the name of the table as well (i.e. cpu_id).  But this has also been a contentious point.

Reply:
I am fine with this. Done.

Reply:
Fair point; I added a comment to the column that it is a primary key and not a processor instruction to read capabilities to help with any confusion.